.circleci/config.yml

```diff
@@ -256,12 +257,7 @@ jobs:
             cd dash-html-components; npm ci && npm run build; rm -rf !(.|..|DESCRIPTION|LICENSE.txt|LICENSE|NAMESPACE|.Rbuildignore|R|man|inst|vignettes|build)
             cd ../dash-core-components; npm ci && npm run build; rm -rf !(.|..|DESCRIPTION|LICENSE.txt|LICENSE|NAMESPACE|.Rbuildignore|R|man|inst|vignettes|build)
             cd ../dash-table; npm ci && npm run build; rm -rf !(.|..|DESCRIPTION|LICENSE.txt|LICENSE|NAMESPACE|.Rbuildignore|R|man|inst|vignettes|build); cd ..
-
-      - run:
-          name: 🔧 fix up dash metadata
-          command: |
-            sudo Rscript dashR/tests/circleci/fixup_metadata.R
```

Comment:
This step is no longer necessary?

Reply:
That's correct. The purpose of [this script](https://github.com/plotly/dashR/blob/master/tests/circleci/fixup_metadata.R) was to clean up versioning errors and update the latest commit hashes for `dcc`, `html`, and `table`. But now that those are included within `dashR` this is no longer necessary. There's some other cleanup we can do for other scripts involving `dashR` as well, but I think that's worth it's own PR. This change however, is necessary to get the tests to run.